qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -299,21 +298,19 @@ def __compose_clifford_seq(
     def __compose_clifford(
         self, left_elem: SequenceElementType, right_elem: SequenceElementType
     ) -> SequenceElementType:
-        if self.num_qubits <= 2:
-            utils = self._cliff_utils
-            return utils.compose_num_with_clifford(
-                left_elem, utils.create_cliff_from_num(right_elem)
-            )
-
+        if self.num_qubits == 1:
+            return compose_1q(left_elem, right_elem)
+        if self.num_qubits == 2:
+            return compose_2q(left_elem, right_elem)
         return left_elem.compose(right_elem)
```

Comment:
Shouldn't this method be part of `CliffordUtils`? Same for ` __adjoint_clifford`